iphone/Classes/TiUIAttributedStringProxy.m

```diff
@@ -67,6 +67,7 @@ -(void)addAttribute:(id)args
 	
 	NSString *attrName = nil;
 	NSString *errorMessage = nil;
+    NSMutableParagraphStyle *paragraphStyle = nil;
```

Comment:
Incorrect indentation. Also, the `paragraphStyle` should stay in the local scope of the switch-case.